LondonBritishLibrary/orient/BLorient4611.xml

```diff
@@ -0,0 +1,274 @@
+                                    <msItem xml:id="ms_i1.2.1.15">
+                                        <locus target="#94r"/>
+                                        <title ref="LIT2272salota"/>
+                                    </msItem>
+                                </msItem>
+                                <msItem xml:id="ms_i1.2.2">
+                                    <locus from="94r" to="97v"/>
+                                    <title ref="LIT2362Songof"/>
+                                    <note>The text is divided into five sections. Indicated by the title: <q xml:lang="gez">መኃልየ፡ መኃልይ፡ ዝውእቱ፡ ዘሰሎሞን።</q></note>
+                                    <msItem xml:id="ms_i1.2.2.1">
+                                        <locus from="94r" to="94v"/>
+                                        <title>First section</title>
+                                        <note>Without title, marked <q xml:lang="gez">፩</q>.</note>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.2.2">
+                                        <locus from="94v" to="95r"/>
+                                        <title>Second section</title>
+                                        <note>Indicated by the title: <q xml:lang="gez">ካልዕ፡ መኃልይ፡ ዘይዜኑ፡ ርደቶ፡ ለዋህድ፡ ወተሰግዎቱ፡ እምድንግል፡</q></note>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.2.3">
+                                        <locus from="95r" to="96r"/>
+                                        <title>Third section</title>
+                                        <note>Indicated by the title: <q xml:lang="gez">ሣልስ፡ መኃልይ፡ ዘይነግር፡ ዕርገተ፡ ትስብእት፡ በ፩ከዊን፡</q></note>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.2.4">
+                                        <locus from="96r" to="97v"/>
+                                        <title>Fourth section</title>
+                                        <note>Indicated by the title: <q xml:lang="gez">ራብዕ፡ መኃልይ፡ ትብሎ፡ መርዓት፡ ለመርዓዊሃ፡</q></note>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.2.5">
+                                        <locus from="97r" to="97v"/>
+                                        <title>Fifth section</title>
+                                        <note>Indicated by the title: <q xml:lang="gez">ኃምስ፡ መኃልይ፡ እምዚአሃ፡ ለቤተ፡ ክርስቲያን፡</q></note>
+                                    </msItem>
+                                </msItem>
+                            </msItem>
+                            <msItem xml:id="ms_i1.3">
+                                <locus from="98ra" to="104vb"/>
+                                <title ref="LIT2509Weddas"/>
+                                <note>With musical notation. Arranged by the days of the week, with Monday's lesson first.</note>
```

Comment:
`<notatedMusic><desc>With musical notation</desc></notatedMusic>`
`<note>Arranged by the days of the week, with Monday's lesson first.</note>`